extension/js/content_scripts/webmail/setup-webmail-content-script.ts

```diff
@@ -329,6 +358,7 @@ export const contentScriptSetupIfVacant = async (webmailSpecific: WebmailSpecifi
       const clientConfiguration = await ClientConfiguration.newInstance(acctEmail);
       await startPullingKeysFromEkm(acctEmail, clientConfiguration, factory, ppEvent);
       await webmailSpecific.start(acctEmail, clientConfiguration, inject, notifications, factory, notifyMurdered);
```

Comment:
why should we wait for completion of pulling the keys before `start`-ing the `webmailSpecific`?
This, of course, leads to less code, but worse UX.
They can be nicely run in parallel, I guess (by passing `() => notifyExpiringKeys(...)` as an argument to `startPulling..` if you like)

Reply:
Updated